ginivision/src/androidTest/java/net/gini/android/vision/test/exif/ExifReaderException.java

```diff
@@ -0,0 +1,14 @@
+package net.gini.android.vision.test.exif;
+
+import android.support.annotation.NonNull;
+
+public class ExifReaderException extends RuntimeException {
```

Comment:
To be able to catch this specific exception. If I would throw a RuntimeException and we would want to catch this specific exception we would have to catch all RuntimeExceptions and filter by message or something:
```
try {
} catch (RuntimeException e) {
    // I now catch all exceptions that extend RuntimeException...
}
```

But with our own subclass we can catch just our exception:
```
try {
} catch (ExifReaderException e) {
    // I catch only the exception I want
}
```